src/constant.js

```diff
@@ -0,0 +1,22 @@
+export const ERROR_MESSAGES = {
+  LENGTH: "[ERROR] 로또 번호는 6개여야 합니다.",
+  DUPLICATED_NUMBER: "[ERROR] 중복되는 숫자는 존재할 수 없습니다.",
+  NOT_IN_RANGE: "[ERROR] 로또 번호는 1부터 45 사이의 숫자여야 합니다.",
+  COAST_WITH_NUMBER: "[ERROR] 구입 금액은 숫자여야 합니다.",
+  COAST_UNIT: (coastUnit) =>
+    `[ERROR] 구입 금액은 ${coastUnit}원 단위여야 합니다.`,
+  COAST_MUST_BE_POSITIVE: "[ERROR] 구입 금액은 양수여야 합니다.",
+};
+
+export const LOTTO_MIN_NUMBER = 1;
+export const LOTTO_MAX_NUMBER = 45;
+
+export const LOTTO_NUMBER_AMOUNT = 6;
+
+export const LOTTO_COAST_UNIT = 1000;
+
+export const LOTTO_FIRST_PRIZE = 2000000000;
+export const LOTTO_SECOND_PRIZE = 30000000;
+export const LOTTO_THIRD_PRIZE = 1500000;
+export const LOTTO_FOURTH_PRIZE = 50000;
+export const LOTTO_FIFTH_PRIZE = 5000;
```

Comment:
```suggestion
export const LOTTO_FIRST_PRIZE = 2_000_000_000;
export const LOTTO_SECOND_PRIZE = 30_000_000;
export const LOTTO_THIRD_PRIZE = 1_500_000;
export const LOTTO_FOURTH_PRIZE = 50_000;
export const LOTTO_FIFTH_PRIZE = 5_000;
```

이런식으로 numeric separator를 이용하면 가독성 측면에서 더 유리할 것 같아요 !

Reply:
숫자 가독성을 고려하지 못했네요. 확실히 자릿수가 나타나도록 separator를 사용하니 보기 좋아요! 좋은 의견 감사해요 :)